packages/flutter_tools/test/commands/drive_test.dart

```diff
@@ -291,6 +291,130 @@ void main() {
         Platform: macOsPlatform,
       });
     });
+
+    group('build arguments', () {
+      String testApp, testFile;
+
+      setUp(() {
+        restoreAppStarter();
+      });
+
+      Future<void> appStarterSetup() async {
+        withMockDevice();
+
+        final MockDeviceLogReader mockDeviceLogReader = MockDeviceLogReader();
+        when(mockDevice.getLogReader()).thenReturn(mockDeviceLogReader);
+        final MockLaunchResult mockLaunchResult = MockLaunchResult();
+        when(mockLaunchResult.started).thenReturn(true);
+        when(mockDevice.startApp(
```

Comment:
I would indent these twice

Reply:
Cool, yeah, I don't like this look anyway :)